content/en/security/guide/security-findings-migration.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Migrating to the New Security Findings Data Model
+description: Learn how to migrate to the unified Security Findings query syntax and data model, which consolidates querying across Cloud Security and Code Security findings.
+private: true
+further_reading:
+- link: "/security/cloud_security_management/"
+  tag: "Documentation"
+  text: "Cloud Security"
+- link: "/security/code_security/"
+  tag: "Documentation"
+  text: "Code Security"
+- link: "/security/application_security/"
+  tag: "Documentation"
+  text: "Application Security"
+---
+
+## Overview
+
+The way Security Findings are queried is changing and it may impact your workflows. You notice changes to how queries are constructed in Datadog. A set of [new features](#new-features) exposing that new schema is also released as a part of the upgrade.
+
+## Required action
+
+If you are using {insert relevant APIs / Terraform resource}, plan to migrate to the new version by {deprecation date}.
+
+Configuration in Datadog is updated automatically, so no action is needed there.
```

Comment:
Can we provide some detail on what "configuration in Datadog" means? How could a user determine what they need to change and what they can leave to us?